build.gradle

```diff
@@ -1833,27 +1848,30 @@ project(':jmh-benchmarks') {
   apply plugin: 'com.github.johnrengelman.shadow'
 
   shadowJar {
-    baseName = 'kafka-jmh-benchmarks-all'
-    classifier = null
```

Comment:
Those changes rename the shadow jar from `kafka-jmh-benchmarks-all.jar` to `kafka-jmh-benchmarks-all-3.0.0-SNAPSHOT-all.jar`.  That breaks `jmh.sh` as the script presumes the name of shadow jar is `kafka-jmh-benchmarks-all.jar` (https://github.com/apache/kafka/blob/trunk/jmh-benchmarks/jmh.sh#L40)

Reply:
Good point, fixed.

Reply:
Could we remove duplicate "all" from name of shadow jar? `kafka-jmh-benchmarks-all-3.0.0-SNAPSHOT-all.jar`

Reply:
Done.